packages/components/atoms/f-popover/src/components/Popover.vue

```diff
@@ -18,15 +18,32 @@ export default {
 
 <style lang="scss" module>
 
+$popover-radius                : 3px;
```

Comment:
Should update this to `2px` to match the new PIE guidelines fro border radius.

Reply:
(can just use `$border-radius` variable)

Reply:
Good point, I just found a popover design in abstract, will check and update

Reply:
Cool – if it's not 2px, make sure you flag it with design, as that's the only border-radius they've defined in the design-tokens atm

Reply:
It is 2px so all good, plus I removed border and updated the shadow according to the latest designs.